roles/kubernetes-apps/csi_driver/cinder/defaults/main.yml

```diff
@@ -15,3 +15,4 @@ cinder_cacert: "{{ lookup('env','OS_CACERT') }}"
 # For now, only Cinder v3 is supported in Cinder CSI driver
 cinder_blockstorage_version: "v3"
 cinder_csi_controller_replicas: 1
+cinder_csi_ignore_volume_az: false
```

Comment:
I wonder what is the point of  having this default overriding the default in `roles/kubespray-defaults/defaults/main.yaml`

Reply:
Thanks. Removed.